src/components/Card/styled/container.js

```diff
@@ -1,20 +1,29 @@
 import styled from 'styled-components';
-import { COLOR_WHITE, COLOR_GRAY_2 } from '../../../styles/colors';
 import { BORDER_RADIUS_1 } from '../../../styles/borderRadius';
-import { SHADOW_4, SHADOW_6 } from '../../../styles/shadows';
 import { MARGIN_LARGE } from '../../../styles/margins';
+import getTheme from '../../../styles/helpers/getTheme';
 
-const StyledContainer = styled.article`
+const StyledContainer = styled.article.attrs(props => {
+    const palette = getTheme(props).palette;
+    const shadow4 = `0 1px 2px 0 ${palette.divider}`;
+    const shadow6 = `0 2px 12px 0 ${palette.divider}`;
+    return {
+        palette,
+        shadow4,
+        shadow6,
+    };
+})`
     position: relative;
-    background-color: ${COLOR_WHITE};
-    border: 0.0625rem solid ${COLOR_GRAY_2};
+    background-color: ${props => props.palette.background.main};
+    color: ${props => props.palette.text.main};
+    border: 0.0625rem solid ${props => props.palette.divider};
```

Comment:
- I think that is not semantic, why we are passing to the border `palette.divider`?
In order to, we have different borders color in various components
e.g the `Input`componet use a different `border-color` than the `Card` component
- Shouldn't we create an entry with the name border?